tool/data_dir_stat.py

```diff
@@ -0,0 +1,30 @@
+import os
+import datetime
+
+from katikati_pylib.logging import logging
+
+log = None
+
+def get_size_and_upload(CRYPTO_TOKEN_PATH, dirpath, project, firebase_client):
+    global log
+    if log is None:
+        log = logging.Logger(__file__, CRYPTO_TOKEN_PATH)
+    dirname = os.path.basename(dirpath)
+    dir_size = get_dir_size(dirpath)
+    entry = {
+        'project': project,
+        'size': round(dir_size, 1)
+    }
+    collection = dirname + '_dir_size_metrics'
```

Comment:
consider keeping the collection name as "dir_size_metrics" and adding the dirname as part of the entry

Reply:
Renamed it to that